<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsoOptions</class>
 <widget class="QDialog" name="IsoOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="IsoFinder_GBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>761</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Finder</string>
   </property>
   <widget class="QPushButton" name="IsoFind_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="IsoFinder_PBar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="dalton_SPbox_2">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.000500000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="dalton_RBtn_2">
       <property name="text">
        <string>Dalton Difference</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="ppm_RBtn_2">
       <property name="text">
        <string>ppm Difference</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="ppm_SPbox_3">
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="IsoOptions_btn">
       <property name="text">
        <string>Iso Pattern Options</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>760</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Properties</string>
   </property>
   <widget class="QGroupBox" name="Option_GBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>365</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>Search Options</string>
    </property>
    <widget class="QRadioButton" name="Pos_RBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Positive Spectra</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Neg_RBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Negative Spectra</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ppm_SPbox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Search_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
    <widget class="QComboBox" name="charge_cBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>69</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="maxCount">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Charge</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ppm Difference</string>
     </property>
    </widget>
    <widget class="QLabel" name="Search_LB">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="ElFinder_PBar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>365</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>Search Elements</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>231</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Element</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Max</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_SearchMethod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Compounds list file</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="files_selectro_Layout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="EC_Open_btn">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EC_line"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="EC_label">
       <property name="text">
        <string>Elements file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Refine_GBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>761</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Refine</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="DBE_CBox">
       <property name="text">
        <string>DBE </string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="OC_max_SBox">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="DBE_min_SBox">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="DBE_max_SBox">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="HC_min_SBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="HC_max_SBox">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="OC_min_SBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="OC_CBox">
       <property name="text">
        <string>O/C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="HC_CBox">
       <property name="text">
        <string>H/C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Refine_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refine</string>
    </property>
   </widget>
   <widget class="QLabel" name="Refine_LB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="Refiner_PBar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
